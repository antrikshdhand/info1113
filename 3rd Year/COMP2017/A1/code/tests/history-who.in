place A1
place B1
place C1
place D1
place E1
who
place F1
place A2
place E3
who
place G4
history
term
