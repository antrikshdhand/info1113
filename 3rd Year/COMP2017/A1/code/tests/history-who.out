W
B
A1B1C1D1E1F1A2E3G4
